client/settings/settings-section.js

```diff
@@ -0,0 +1,13 @@
+/** @format */
+
+const SettingsSection = ( { title, description, children } ) => (
```

Comment:
I like that this is now its own file! However, the styles for `settings-manager__section`, `settings-manager__section-details` and `settings-manager__controls` are still in this file: `client/settings/style.scss`.
Should `SettingsSection` have its own separate styles file?

Reply:
I did this and changed the class name from `settings-manager__section` to `settings-section`.